src/Facebook/FacebookBatchRequest.php

```diff
@@ -62,20 +62,21 @@ public function __construct(FacebookApp $app = null, array $requests = [], $acce
     }
 
     /**
-     * A a new request to the array.
+     * Adds a new request to the array.
      *
      * @param FacebookRequest|array $request
      * @param string|null           $name
+     * @param string[]              $options The batch options e.g. 'omit_response_on_success'
      *
      * @return FacebookBatchRequest
      *
      * @throws \InvalidArgumentException
      */
-    public function add($request, $name = null)
+    public function add($request, $name = null, array $options = [])
     {
         if (is_array($request)) {
             foreach ($request as $key => $req) {
-                $this->add($req, $key);
+                $this->add($req, strval($key));
```

Comment:
@teldosas Why did we add `strval()` here?
@yguedidi This is a BC break when you're expecting numeric array keys since they will be returned as strings now.